_entities.md

```diff
@@ -145,6 +145,38 @@ bid      | The current bid price, or the price we quote when buying the asset.
 currency | The currency that is used in the `ask` and `bid` prices.
 pair     | The currency pair AB represents moving from A to B.
 
+## Phone Object
+
+> An example phone encoded in JSON looks like this:
+
+```
+{
+  "e164Masked": "+XXXXXXXXX04",
+  "id": "1d78aeb5-43ac-4ee8-8d28-1291b5d8355c",
+  "internationalMasked": "+X XXX-XXX-XX04",
+  "nationalMasked": "(XXX) XXX-XX04",
+  "primary": true,
+  "verified": true
+}
+```
+
+A phone number is required for registration and, when enabled, provides additional security to the user's account through the use of Two-Factor Authentication (2FA).
```

Comment:
Phones and 2FA are now separated. Also, please check if a phone really is required for registration, I don't know if that has changed.

Reply:
Yeah, it doesn't seem to be required when registering an account. I'll remove this sentence altogether; thanks for the heads-up.

Reply:
[Updated](https://github.com/uphold/docs/compare/f0e403605b7ae21f7615b4ee69941a80be500870..a0518dcbc93dc0cccc35e7fc6bcdcc1b1c571915).